artwork/logo/coala_color.svg

```diff
@@ -331,15 +331,15 @@
      inkscape:pageopacity="0.0"
      inkscape:pageshadow="2"
      inkscape:zoom="11.313709"
-     inkscape:cx="7.3668075"
+     inkscape:cx="-13.537036"
      inkscape:cy="24.475906"
      inkscape:current-layer="layer3"
      showgrid="false"
      inkscape:grid-bbox="true"
      inkscape:document-units="px"
      inkscape:snap-global="false"
      inkscape:window-width="1366"
-     inkscape:window-height="709"
+     inkscape:window-height="718"
```

Comment:
Window height may just refer to the height of the canvas of the editor, it may not impact the end image at all.

Reply:
ya, that sounds right